### ISSUE TITLE
`python310Packages.nbconvert` build failure on x86_64-linux as of `a093c35b`

### ISSUE THREAD

**Issue by samuela** [NixOS/nixpkgs#186871]
## Issue description
Build of `python310Packages.nbconvert` failed on x86_64-linux as of a093c35beee0a64ad68312d747a4c49eaa5b1105. This is currently breaking `python3Packages.einops`.

```
error: builder for '/nix/store/4277frbm7rmf2pw9nk3fxnv4zbn5w40w-python3.10-nbconvert-6.5.0.drv' failed with exit code 1;
       last 10 log lines:
       > Requirement already satisfied: jupyter-core>=4.7 in /nix/store/ap0g73di2873pj7m28i6ww8zn0cnvihg-python3.10-jupyter_core-4.9.2/lib/python3.10/site-packages (from nbconvert==6.5.0) (4.9.2)
       > Requirement already satisfied: bleach in /nix/store/1agcqr28acfw6cf1cag4da7f9rx685dp-python3.10-bleach-5.0.1/lib/python3.10/site-packages (from nbconvert==6.5.0) (5.0.1)
       > Requirement already satisfied: nbformat>=5.1 in /nix/store/h2k2q522y8c48szz39wgipc673pf62vf-python3.10-nbformat-5.4.0/lib/python3.10/site-packages (from nbconvert==6.5.0) (5.4.0)
       > Requirement already satisfied: pygments>=2.4.1 in /nix/store/czicm46iryrk0rssz4zs8rn788x4ni42-python3.10-pygments-2.12.0/lib/python3.10/site-packages (from nbconvert==6.5.0) (2.12.0)
       > Requirement already satisfied: entrypoints>=0.2.2 in /nix/store/6r1vdaz2vj5gzlnb9spz8inv0r7gc3iq-python3.10-entrypoints-0.4/lib/python3.10/site-packages (from nbconvert==6.5.0) (0.4)
       > Requirement already satisfied: packaging in /nix/store/rcvz5ywnm8fla013hqsg01imz2i41k7p-python3.10-packaging-21.3/lib/python3.10/site-packages (from nbconvert==6.5.0) (21.3)
       > Requirement already satisfied: jinja2>=3.0 in /nix/store/nd6636vpb09q2f9p79a5w3kdhv1qbw2j-python3.10-Jinja2-3.1.2/lib/python3.10/site-packages (from nbconvert==6.5.0) (3.1.2)
       > ERROR: Could not find a version that satisfies the requirement mistune<2,>=0.8.1 (from nbconvert) (from versions: none)
       > ERROR: No matching distribution found for mistune<2,>=0.8.1
       > 
       For full logs, run 'nix log /nix/store/4277frbm7rmf2pw9nk3fxnv4zbn5w40w-python3.10-nbconvert-6.5.0.drv'.
error: 1 dependencies of derivation '/nix/store/jpb7bh0g1j156vvziclksfxmrzs9fc25-python3.10-einops-0.4.1.drv' failed to build
```

[full build log](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2864366564)

python3Packages.einops maintainers cc: @yl3dy
python310Packages.nbconvert maintainers cc: @FRidh
Other cc: n/a

### Steps to reproduce
1. Checkout nixpkgs at commit a093c35beee0a64ad68312d747a4c49eaa5b1105
2. Run `nix-build -A python310Packages.nbconvert`

## Technical details
```
 - system: `"x86_64-linux"`
 - host os: `Linux 5.15.0-1014-azure, Ubuntu, 20.04.4 LTS (Focal Fossa), nobuild`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.10.3`
 - nixpkgs: `/nix/store/xbmc7mwndkbykkcdkgwvl2i8dx43db6x-source`
```

## Misc.
This issue was automatically generated by [nixpkgs-upkeep](https://github.com/samuela/nixpkgs-upkeep).
- [CI workflow](https://github.com/samuela/nixpkgs-upkeep/actions/runs/2864366564) that created this issue.
- Internal tags: -5805513848243597698 3278788159294963055

**Comment by virchau13** [comment]
Looks like this is due to https://github.com/NixOS/nixpkgs/commit/243053e521dc90b825a17bf5ee9f5dfe7f56af84. `mistune` was updated from 0.8.4 to 2.0.4, which doesn't satisfy `nbconvert`'s requirement of `mistune<2,>=0.8.1`.

**Comment by virchau13** [comment]
Ah, seems it's going to be solved by #186804.
